_posts/2016/2016-07-04-ep6-offsite.md

```diff
@@ -0,0 +1,55 @@
+To make best use of the limited time we held some workshops on parallel tracks.
+
+Of interest to everyone was the "JavaScript in epages 6" session chaired by Markus. He explained in detail how different JavaScript frameworks are integrated in epages 6 and what to consider when writing JavaScript in this environment. Using live coding he demonstrated how the template engine, localization and JavaScript compression work.
+
+{% image blog/blog-ep6-frontend-workshop.jpg 50% left %}
+
+Oliver explained selected [OWASP threats](https://www.owasp.org/index.php/Cheat_Sheets). Afterwards we tried to exploit our software. While we identified some problems, we also concluded that epages 6 is pretty resilient against the most common attacks: injections and cross-site scripting.
+
+Peter outlined our current development pipeline process used primarily for QA and collaboration on more complex features.
+Additionally he clarified the different build  and configuration setups on our team VMs.
+
+On the second day we started with the parallel sessions: UI development and tax calculation in epages 6.
+For the frontend developers and designers Björn presented the various HTML template sections of a typical epages 6 webpage. In addition to that he documented the directory and file structure used for styling as well as the style guide itself.
+The backend folks attended the tax calculation lecture by Michael. He explained in detail how to alter tax settings in epages 6 and which pitfalls to consider when customers and merchants from different countries do business together.
+
+Last but not least Tobias emphasized unit testing and best practices like the [Test Pyramid](http://martinfowler.com/bliki/TestPyramid.html). Using multiple code examples he demonstrated how mocking can be used for epages 6 integration tests.
+
+## Bow and Arrow Action
+
+{% image blog/blog-ep6offsite-bow.jpg 50% right %}
+
+After one and a half day of intense workshops in the Creative Room of the Volkshochschule Jena, we had our recreational team event on Wednesday afternoon. The [archery site](http://camburger-bogenschuetzen.de/) was located in the forest area in Camburg.
+
+Before starting our tour through the forest, we had some training shots. Equipped with bows, arrows, quivers and protection gear, we were prepared for small and big game! We picked the "lesser" route, which resulted in shooting at around 27 animal models in three hours. Besides the typical forest animals we hunted gigantic turtles, cobras and even dinosaurs! :)
+
+When the sun had set and it was hard to spot any further targets, we gathered for a barbecue to end the day, share our hunting stories and compare our scores.
+
+## Back at the Office
+
+It was an awesome team event! We learned a lot during the workshops, prepared ourselves for the upcoming team changes and embraced the nature near Camburg.
```

Comment:
It was an awesome team event! We learned a lot during the workshops, prepared ourselves for the upcoming team changes and embraced the nature near Camburg. > It was an awesome team event! We learned a lot during the workshops, prepared ourselves for the upcoming team changes, and embraced the nature near Camburg. (comma after changes)